### DISCUSSION TITLE
Installation on Raspberry1

### DISCUSSION THREAD

**Discussion by HendrikL13** [john30/ebusd#678] [Q&A]
I have compiled and installed ebusd on raspberry1.
wiki https://github.com/john30/ebusd/wiki/1.-Build-and-install#installation tells us the installation command:
make install-strip
make: *** No rule to make target 'install-strip'.  Stop.

After reading the Makefile (generated by cmake) I found out that the correct command is
make install/strip
So I think the command given in the wiki is incorrect.

**Comment by john30** [comment]
if you follow that page and use ./autogen.sh it should be fine

**Reply by G8GYW** [reply]
What OS are you running on your Pi?

Mike

**Reply by HendrikL13** [reply]
not the latest.
It is raspbian buster.

Does that explain the diff between install-strip vs install/strip ?
